roboteam_ai/src/utilities/RobotDealer.h

```diff
@@ -27,50 +31,55 @@ class RobotDealer {
 
     private:
 
-        static std::map<std::string, std::set<std::pair<int, std::string>>> robotOwners;
+        std::map<std::string, std::set<std::pair<int, std::string>>> robotOwners;
```

Comment:
I don't think we should change robotdealer as much in this PR. Keep the focus on World